lib/elixir/lib/registry.ex

```diff
@@ -444,6 +444,11 @@ defmodule Registry do
   value or tuple element, while :"$1" can be used to temporarily assign part
   of pattern to a variable for a subsequent comparison.
 
+  It is possible to pass list of guard conditions for more precise matching.
+  Each guard is a tuple, which describes check that should be passed by assigned part of pattern.
+  For example :"$1" > 1 guard condition would be expressed as {:>, :"$1", 1} tuple.
+  Please note that guard conditions will work only for assigned variables like :"$1", :"$2", etc.
```

Comment:
Yeah, the issue is that `:"$_"` is for the whole term (`{key, {pid, value}}`) and not just `value`.